src/main/java/uk/gov/hmcts/divorce/notification/NotificationDispatcher.java

```diff
@@ -34,18 +34,7 @@ public void sendNOC(final ApplicantNotification applicantNotification,
                         final CaseData caseData, final CaseData previousCaseData, final Long caseId,
                         boolean isApplicant1, NoticeType noticeType) {
         if (noticeType == NoticeType.NEW_DIGITAL_SOLICITOR_NEW_ORG) {
-            if (isApplicant1) {
-                if (StringUtils.isNotEmpty(caseData.getApplicant1().getEmail())) {
-                    applicantNotification.sendToApplicant1(caseData, caseId);
-                }
-                applicantNotification.sendToApplicant1Solicitor(caseData, caseId);
-
-            } else {
-                if (StringUtils.isNotEmpty(caseData.getApplicant2().getEmail())) {
-                    applicantNotification.sendToApplicant2(caseData, caseId);
-                }
-                applicantNotification.sendToApplicant2Solicitor(caseData, caseId);
-            }
+            sendNotification(isApplicant1, caseData, caseId, applicantNotification);
```

Comment:
**Nitpick**: The method name seems a bit ambiguous because we also send notifications in the if statement below. How about something like `sendRepresentationGrantedNotifications`?